nipoppy/trackers/run_tracker.py

```diff
@@ -20,79 +22,93 @@
 }
 BIDS_PIPES = ["mriqc","fmriprep", "tractoflow"]
 
-def run(global_config_file, dash_schema_file, pipelines, run_id=1):
+def run(global_configs, dash_schema_file, pipelines, session_id, run_id=1, logger=None):
     """ driver code running pipeline specific trackers
     """
+    session = f"ses-{session_id}"    
+    DATASET_ROOT = global_configs["DATASET_ROOT"]
+
+    # logging
+    log_dir = f"{DATASET_ROOT}/scratch/logs/"
+    if logger is None:
+        log_file = f"{log_dir}/mriqc.log"
+        logger = my_logger.get_logger(log_file)
+
+    logger.info(f"Tracking pipelines: {pipelines} for session: {session}")
 
     proc_status_dfs = [] # list of dataframes
     for pipeline in pipelines:
-        pipe_tracker = tracker(global_config_file, dash_schema_file, pipeline) 
+        pipe_tracker = tracker(global_configs, dash_schema_file, pipeline) 
             
-        dataset_root, session_ids, version = pipe_tracker.get_global_configs()
+        DATASET_ROOT, session_ids, version = pipe_tracker.get_global_configs()
         schema = pipe_tracker.get_dash_schema()
         tracker_configs = pipeline_tracker_config_dict[pipeline]
 
-        manifest = f"{dataset_root}/tabular/manifest.csv"
-        manifest_df = pd.read_csv(manifest)
-        participants = manifest_df[~manifest_df["bids_id"].isna()]["bids_id"].drop_duplicates().astype(str).str.strip().values
-        n_participants = len(participants)
+        # TODO
+        # Check if session_id belongs to session_ids from global configs
 
-        print("-"*50)
-        print(f"pipeline: {pipeline}, version: {version}")
-        print(f"n_participants: {n_participants}, session_ids: {session_ids}")
-        print("-"*50)
+        # Grab BIDS participants from the doughnut
+        doughnut_file = f"{DATASET_ROOT}/scratch/raw_dicom/doughnut.csv"
+        doughnut_df = pd.read_csv(doughnut_file)
+        doughnut_df["converted"] = doughnut_df["converted"].astype(bool)
```

Comment:
Can we use [`nipoppy.workflow.utils.load_status()`](https://github.com/michellewang/nipoppy/blob/92/schema/nipoppy/workflow/utils.py#L95-L109) for this?

Reply:
Good idea, done!